### ISSUE TITLE
warning message when type="color"

### ISSUE THREAD

**Issue by shosaco** [mtennekes/treemap#37]
There is a data.table warning message when using `type = color`. Reproducible example:

```
mtcars %>% head %>% 
   mutate(color = "red") %>% 
   treemap(index = "cyl", vSize = "cyl", vColor = "color", type = "color")

Warning message:
In `[.data.table`(dtfDT, , `:=`("c", fact), with = FALSE) :
  with=FALSE ignored, it isn't needed when using :=. See ?':=' for examples.
```

**Comment by shosaco** [comment]
I just noticed that this was issue #34 and should have been fixed in verison 2.4.2. However, it is not. The old code (line 499f in `treemap.R`) is still included in the [CRAN release](https://cran.r-project.org/src/contrib/treemap_2.4-2.tar.gz).

**Comment by mtennekes** [comment]
Thanks for reporting this. I found out that #34 is fixed _after_ 2.4.2 is released. See https://github.com/mtennekes/treemap/commits/master
